packages/logs/src/domain/logger.ts

```diff
@@ -21,10 +21,15 @@ export interface LogsMessage {
 }
 
 export const StatusType = {
+  OK: 'OK',
```

Comment:
**🥜 nitpick:** ‏I know this is like this in the documentation, but for consistency I would use lowercase for `ok` (key and value) similarly to other variants.